### PR TITLE
CB-12810 add assertion to multiaz case

### PR DESCRIPTION
**Pull request by afarsang**
See detailed description in the commit message.

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/assertion/distrox/AvailabilityZoneAssertion.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import com.sequenceiq.it.cloudbreak.assertion.Assertion;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+
+public class AvailabilityZoneAssertion implements Assertion<DistroXTestDto, CloudbreakClient> {
+
+    @Override
+    public DistroXTestDto doAssertion(TestContext testContext, DistroXTestDto testDto, CloudbreakClient client) throws Exception {
+        if (testDto.getCloudPlatform() != CloudPlatform.AWS) {
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/assertion/distrox/AvailabilityZoneAssertion.java:23]: do we need to also check the variant?

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXCustomTest.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    @Test(dataProvider = TEST_CONTEXT)
+    @UseSpotInstances
+    @Description(
+            given = "there is a running cloudbreak and existin environment, distrox, datalake",
+            when = "a valid DistroX",
+            then = "DistroX cluster is mutliaz")
+    public void testCreateDistroXWithEncryptedVolume(TestContext testContext) {
+
+        testContext
+                .given(CredentialTestDto.class).withName("aws-test-ee4d66dfa2")
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXCustomTest.java:65]: is this ok with fix name

**Review by doktoric — CHANGES_REQUESTED** [review]

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXCustomTest.java (hunk 2 above)
Comment (afarsang) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXCustomTest.java:65]: ok

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/assertion/distrox/AvailabilityZoneAssertion.java (hunk 1 above)
Comment (afarsang) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/assertion/distrox/AvailabilityZoneAssertion.java:23]: i haven't seen it in the response.. I chose to check the metadata availibitily zone size instead

**Review by doktoric — APPROVED** [review]

**Pull request merged**